src/Templates/form.twig

```diff
@@ -37,7 +37,7 @@
 {{ parent() }}
 {% include 'parts/table-footer.twig' %}
 <script type="text/javascript" src="{{baseurl}}js/vendor/jquery.validate.min.js"></script>
-<script type="text/javascript" src="{{baseurl}}js/vendor/tinymce.min.js"></script>
+<script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
```

Comment:
A local copy would be preferable to me also, mostly because MyRadio currently does a very good job at having no external dependencies, which would be nice to keep for all the reasons above.